### PR TITLE
Remove old code. Close #19

### LINKED ISSUE #19: Source File Pruning
PR #16 moved the functionality of the PhotoLib view and Settings view to the Main view. We should remove these files. The PageViewController is also no longer needed because the app does not support multiple pages.

### PR DESCRIPTION
**Pull request by ianstenbit**

### CODE REVIEW

**Comment by ianstenbit** [comment]
Please do not merge this until after #25 is closed.

**Review by jakecarlson1 — APPROVED** [review]
Goodbye redundant files, hello cleaner code base.

**Pull request merged**
